homeassistant/components/webostv/media_player.py

```diff
@@ -59,7 +59,7 @@
 
 MIN_TIME_BETWEEN_SCANS = timedelta(seconds=10)
 MIN_TIME_BETWEEN_FORCED_SCANS = timedelta(seconds=1)
-
+SCAN_INTERVAL = timedelta(seconds=10)
```

Comment:
@bendavid what do you think about this change? The polling update is just used to check the connection, right?

Reply:
Yes that's right, and in fact it's only relevant when the device is off, so indeed this looks fine.